src/components/MapView/index.js

```diff
@@ -48,13 +61,62 @@ class MapView extends Component
                         longitude={parseFloat(property.longitude)}
                         latitude= {parseFloat(property.latitude)}
                         key={propertyId}>
-                        <span className="icon has-text-info">
+                        <span className="icon has-text-info"
+                        onClick = {(e) => {
+                            e.preventDefault();
+                            this.setState({showPopup: true, popupProperty: property});
+                            
+
```

Comment:
remove this empty spaces

Reply:
I've done everything else, just one thing:

React router uninstall karke reinstall kiya to same version hi hua, 5.1.2. Is there a particular version that you want me to npm i?

Reply:
reinstall nahi karna tha bhai , react router uninstall karna tha